packages/block-library/src/navigation-submenu/index.php

```diff
@@ -188,6 +188,12 @@ function render_block_core_navigation_submenu( $attributes, $content, $block ) {
 		);
 	}
 
+	$arialabel = sprintf(
+		/* translators: button label name */
+		__( '%s submenu' ),
```

Comment:
The translator comment is not really helpful as it doesn't explain what the `%s` placeholder is for. It's also not so easy to translate. My suggestion:

```suggestion
		/* translators: Accessibility text. %s: Parent page title. */
		__( 'Submenu of %s' ),
```

Reply:
Was struggling with it myself, thanks for suggestion and the review! Updated both cases to the same now.